[2023-02-17T08:22:28.808+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_scrape_from_wiki.CONNECTION_SUCCESS manual__2023-02-17T08:22:26.707038+00:00 [queued]>
[2023-02-17T08:22:28.819+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_scrape_from_wiki.CONNECTION_SUCCESS manual__2023-02-17T08:22:26.707038+00:00 [queued]>
[2023-02-17T08:22:28.820+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-02-17T08:22:28.821+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-02-17T08:22:28.822+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-02-17T08:22:28.837+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): CONNECTION_SUCCESS> on 2023-02-17 08:22:26.707038+00:00
[2023-02-17T08:22:28.844+0000] {standard_task_runner.py:55} INFO - Started process 244 to run task
[2023-02-17T08:22:28.849+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_scrape_from_wiki', 'CONNECTION_SUCCESS', 'manual__2023-02-17T08:22:26.707038+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/dag_scrape_specific.py', '--cfg-path', '/tmp/tmpfsr3k95b']
[2023-02-17T08:22:28.852+0000] {standard_task_runner.py:83} INFO - Job 134: Subtask CONNECTION_SUCCESS
[2023-02-17T08:22:29.140+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_scrape_from_wiki.CONNECTION_SUCCESS manual__2023-02-17T08:22:26.707038+00:00 [running]> on host 55b685598511
[2023-02-17T08:22:29.211+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=test@test.com
AIRFLOW_CTX_DAG_OWNER=test
AIRFLOW_CTX_DAG_ID=dag_scrape_from_wiki
AIRFLOW_CTX_TASK_ID=CONNECTION_SUCCESS
AIRFLOW_CTX_EXECUTION_DATE=2023-02-17T08:22:26.707038+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-17T08:22:26.707038+00:00
[2023-02-17T08:22:29.213+0000] {connection.py:285} INFO - Snowflake Connector for Python Version: 3.0.0, Python Version: 3.7.15, Platform: Linux-5.15.49-linuxkit-x86_64-with-debian-11.5
[2023-02-17T08:22:29.215+0000] {connection.py:975} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2023-02-17T08:22:29.216+0000] {connection.py:992} INFO - Setting use_openssl_only mode to False
[2023-02-17T08:22:30.029+0000] {cursor.py:727} INFO - query: [SELECT current_version()]
[2023-02-17T08:22:30.157+0000] {cursor.py:740} INFO - query execution done
[2023-02-17T08:22:30.160+0000] {cursor.py:879} INFO - Number of results in first chunk: 1
[2023-02-17T08:22:30.162+0000] {logging_mixin.py:137} INFO - 7.5.1
[2023-02-17T08:22:30.164+0000] {logging_mixin.py:137} INFO - Successfully connected to Snowflake Database
[2023-02-17T08:22:30.166+0000] {connection.py:581} INFO - closed
[2023-02-17T08:22:30.256+0000] {connection.py:584} INFO - No async queries seem to be running, deleting session
[2023-02-17T08:22:30.409+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-02-17T08:22:30.434+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_scrape_from_wiki, task_id=CONNECTION_SUCCESS, execution_date=20230217T082226, start_date=20230217T082228, end_date=20230217T082230
[2023-02-17T08:22:30.510+0000] {local_task_job.py:159} INFO - Task exited with return code 0
